### PR TITLE
[9.0][IMP] Move from pyBarcode

### PR DESCRIPTION
**Pull request by lasley**
* pyBarcode is no longer maintained, switch to viivakoodi fork (https://bitbucket.org/whitie/python-barcode/issues/16/pypi-08-release-request#comment-33978213)

The owner of the fork is active & responsive, thus I recommend we move to that repo. The change is transparent, but the code is slightly better, Travis tested, and will only improve over time. The other repo is basically the opposite of those points. 

My issues/PRs were handled within an hour:

* https://github.com/kxepal/viivakoodi/issues/6
* https://github.com/kxepal/viivakoodi/pull/5

Vs a series of PRs and issues that are not responded to for months/years:
* https://bitbucket.org/whitie/python-barcode/pull-requests/
* https://bitbucket.org/whitie/python-barcode/issues?status=new&status=open

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi, @lasley, thanks for your work on pybarcode / viivakoodi.
Except that the name of this new lib is unpronounceable for most people ;-) I see no problem to switch to this new lib.

* indeed last commit on pybarcode was 4 months ago.
* there are not a lot of commits on viivakoodi, but it seems to be more reactive
* Tested on runbot and it works.

I did not analyzed the diff between the two lib, but for me LGTM. :+1: 

@gurneyalex, @OCA/community-maintainers do you have any PoV about that change ? 

kind regards.

**Review by legalsylvain — APPROVED** [review]

**Comment by lasley** [comment]
FYI I am now a maintainer of the viivakoodi lib, so I can now vouch for its reactivity first hand. I still can't pronounce it though, although it's kind of fun trying!

I will add the notes to requirements and Readme in both PRs as you rightfully requested in https://github.com/OCA/stock-logistics-barcode/pull/65#issuecomment-275641702

File: barcodes_generator_abstract/README.rst [hunk 1]
```diff
@@ -31,10 +31,11 @@ With this module, it is possible to:
 Installation
 ============
 
-This module use an extra python library named 'pyBarcode' you should install
-to make barcode generation works properly.
+This module use an extra python library named 'viivakoodi' you should install
```
Comment (tedsalmon) [inline barcodes_generator_abstract/README.rst:34]: `s/use/uses`

File: barcodes_generator_abstract/README.rst [hunk 2]
```diff
@@ -31,10 +31,11 @@ With this module, it is possible to:
...
+to make barcode generation works properly. 'viivakoodi' is a more active for of
```
Comment (tedsalmon) [inline barcodes_generator_abstract/README.rst:35]: `s/works/work`

`s/is a more active for of/is a more active fork of`

**Review by tedsalmon — APPROVED** [review]
LGTM but the README isn't entirely legible. See review below. 👍

**Comment by gurneyalex** [comment]
@lasley don't we need a change in the external dependencies in the manifest?

**Review by legalsylvain — CHANGES_REQUESTED** [review]
+1 for @gurneyalex remark. need to change the name of the lib in the manifest file.

**Comment by lasley** [comment]
That's not necessary because the manifest file is based on the library namespace. In this case viivakoodi is a fork of pyBarcode, both of which provide the `barcode` namespace 👍

**Comment by gurneyalex** [comment]
I just restarted the build which failed because of github errors.

**Comment by lasley** [comment]
Woot build 🍏 !

**Review by legalsylvain — APPROVED** [review]

**Comment by legalsylvain** [comment]
Thanks  !

**Pull request merged**